docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_basic_information.yml

```diff
@@ -140,9 +140,12 @@ code: |
   else:
     school.recipient.name = school.special_education_director_or_coordinator.name
     school.recipient.address = school.special_education_director_or_coordinator.address
-    if not knows_special_education_director_or_coordinator_name:
-      school.special_education_coordinator.name = "Special Education Coordinator"
-  school.recipient.name
+  if knows_special_education_director_or_coordinator_name:  
+    school.special_education_director_or_coordinator.name.first
+    school.special_education_director_or_coordinator.name.last
+  else:
+    school.special_education_director_or_coordinator.name.first = "Special Education"
```

Comment:
Change this to `school.special_education_director_or_coordinator.name = "Special Education Coordinator or Director"`